apps/src/redux/isRunning.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This duck module tracks whether the current app is running or not.
+ */
+var TOGGLE_RUNNING = 'isRunning/TOGGLE_RUNNING';
+
+module.exports.default = function (state, action) {
+  var nextState = state || false;
+
+  if (action.type === TOGGLE_RUNNING) {
+    nextState = action.isRunning;
```

Comment:
Interesting thought.

I do wonder though if doing so would just create an extra level of indirection/confusion. I would advocate for leaving things as they are right now, and if we find we're  (a) creating a lot of these (which seems probable) and (b) dont like how much code we're right for them, we go with the creator approach.